src/components/Button/Button.module.css

```diff
@@ -0,0 +1,4 @@
+.ghost {
```

Comment:
are classes namespaced by component? or does this class apply to the whole DOM? If so I'd add a specific prefix to all of the classes we are going to define here